Endzone.Umbraco.Extensions/PublishedContentExtensions/Media.cs

```diff
@@ -105,26 +105,31 @@ public static IHtmlString ShowImages(this IPublishedContent item, string propert
         }
 
         /// <summary>
-        /// Works for MultipleMediaPicker where the image media is of type Image Cropper(as opposed to upload).
+        /// Works for MultipleMediaPicker where the image media is of type Image Cropper(as opposed to upload). Shows the image's urls in their cropped version.
         /// </summary>
         /// <param name="item"></param>
         /// <param name="cropAlias"></param>
         /// <param name="property"></param>
         /// <param name="recurse"></param>
+        /// <param name="prepend"></param>
         /// <param name="append"></param>
         /// <returns></returns>
-        public static IHtmlString ShowImageUrlCropped(this IPublishedContent item, string cropAlias, string property = "image", bool recurse = false, string append = "")
```

Comment:
@filipesousa Yes, but any sites that use the old method `ShowImageUrlCropped` will be broken with the new changes, when the nuget package is updated on those sites.